api/testdata/complete-scenarios/test5.json

```diff
@@ -16654,7 +16654,7 @@
                   "DivorceLocation": {
                     "type": "location",
                     "props": {
-                      "layout": "US City, State, International city",
+                      "layout": "US City, State, Zipcode International city",
```

Comment:
This test data change originated back with https://github.com/18F/e-QIP-prototype/commit/c19617bebff7d29e63f9481c1227babef1a08683 and https://github.com/18F/e-QIP-prototype/pull/833

In that PR some other test data sets were created with the same layout (`"US City, State, International city"`) – do they need to be updated as well with the `Zipcode` variant?

Reply:
Yeah, good catch. Updated those references as well.